### PR TITLE
NIFI-9787: Increasing visibility of selected slickgrid row with 1px b…

### PR DESCRIPTION
**Pull request by jocovo**
…orders on the top and bottom plus a slightly darker yellow background-color value.

Thank you for submitting a contribution to Apache NiFi.

Please provide a short description of the PR here:

#### Description of PR

_Enables easier visibility of selected table rows, implementing WCAG 2.1 success criteria "[Ensuring that additional visual cues are available when text color differences are used to convey information](https://www.w3.org/WAI/WCAG21/Techniques/general/G182)"; fixes ticket [NIFI-9787](https://issues.apache.org/jira/browse/NIFI-9787)._

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [X] Does your PR title start with **NIFI-XXXX** where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [X] Has your PR been rebased against the latest commit within the target branch (typically `main`)?

- [X] Is your initial contribution a single, squashed commit? _Additional commits in response to PR reviewer feedback should be made on this branch and pushed to allow change tracking. Do not `squash` or use `--force` when pushing to allow for clean monitoring of changes._

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via `mvn -Pcontrib-check clean install` at the root `nifi` folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] Have you verified that the full build is successful on JDK 8?
- [ ] Have you verified that the full build is successful on JDK 11?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the `LICENSE` file, including the main `LICENSE` file under `nifi-assembly`?
- [ ] If applicable, have you updated the `NOTICE` file, including the main `NOTICE` file found under `nifi-assembly`?
- [ ] If adding new Properties, have you added `.displayName` in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by jocovo** [comment]
For those less familiar with the UI code, there are some not-readily-apparent things about my proposed changes I'd like to explain. First thing to know is that the UI uses the SlickGrid library to render the tables, which has two behaviors of note:
- when a component designed to filter selectable results from a table is rendered, the first row is marked as "active"
- when a row is marked as "active", SlickGrid will mark that row's cells as "selected"
The next thing to know is that most of the elements in the UI are rendered with absolute positioning, which causes some headaches when making display changes to lists of elements. For example, when trying to get the top border to display for the first row in the table, modifying the _row's_ margin-top offset instead of the padding of the _cells_ causes SlickGrid's "getCellFromPoint()"'s offset math to not be able to find the cells it is attempting to apply the "selected" class to.
The borders are applied to the cells for these reasons, which means the padding-top needs to account for the borders to keep the contents of the cells from visually moving.

File: nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/slick-nifi-theme.css [hunk 1]
```diff
@@ -126,7 +129,7 @@
     border-top: 0;
     border-left: 1px solid #c7d2d7;
     border-bottom: 1px solid #c7d2d7;
-    padding: 0 2px 6px 10px;
+    padding: 1 2px 5px 10px;
```
Comment (sardell) [inline nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/slick-nifi-theme.css:132]: It looks like your padding-top property needs a unit of measurement (ex. 1px).

Comment (jocovo) [reply nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/slick-nifi-theme.css]: Good catch. I've added 'px' and pushed the change.

**Comment by jocovo** [comment]
@sardell Everything looks good on my end. Do you approve?

**Comment by sardell** [comment]
@jocovo This does look good to me. +1 (non-binding).

**Comment by jocovo** [comment]
Thanks for reviewing @sardell ! Can you please remove yourself as the reviewer?

**Comment by sardell** [comment]
@jocovo I'm actually missing the gear icon in the reviewers section and am unable to remove myself (not sure if I just don't have the right permissions for this project). I'm not sure what is resolved by removing me as a reviewer but others can review as well. 

@mtien-apache might have a moment to review and get this merged in.

**Comment by jocovo** [comment]
@sardell Ohhh, okay, I wasn't sure if it would be picked up by others if you were still assigned as the reviewer. Now I know it's not "the" reviewer but "a" reviewer. Thanks.

**Comment by mtien-apache** [comment]
Reviewing...

**Review by mtien-apache — APPROVED** [review]
+1 Looks good! Verified the changes are applied and work well in places where configurations have table rows. Thanks for your contribution, @jocovo!

**Pull request merged**